R/write_checklist.R

```diff
@@ -1,5 +1,18 @@
 #' Write a check list with allowed issues in the source code
 #'
+#' First run `x <- checklist::check_package()` or
```

Comment:
Here I would start with one sentence explaining the main goal of this function: composing a list of warnings or notes that will be allowed and not result in a failed check.
And then I would make clear somehow that you give an explanation of the different steps to be followed to realise this.  Maybe this belongs to the @details section rather than the @description section?  Maybe you want to use ticks for the different steps?

Reply:
done in 9bd6e074ee69752b77af79a0fcf8eca953dd60de